specification/cognitiveservices/Language.AnalyzeConversations-authoring/main.tsp

```diff
@@ -47,6 +47,12 @@ enum Versions {
   @useDependency(Azure.Core.Versions.v1_0_Preview_2)
   v2023_04_01: "2023-04-01",
 
+  /**
+   * The 2023-04-15-preview API version.
+   */
+  @useDependency(Azure.Core.Versions.v1_0_Preview_2)
+  v2023_04_15_preview: "2023-04-15-preview",
```

Comment:
If you're trying to remove previews, why not bump `@added` etc., to the subsequent stable version release?